app/controllers/admin/auctions_controller.rb

```diff
@@ -3,7 +3,7 @@ class AuctionsController < ApplicationController
     before_filter :require_admin
 
     def index
-      @auctions = Auction.all.map { |auction| Presenter::AdminAuction.new(auction) }
+      @auctions = Auction.all.map { |auction| AdminAuctionPresenter.new(auction) }
```

Comment:
This naming style is closer to what I usually see and closer to how rails does naming, eg  `app/helpers/application_helper.rb` using name `ApplicationHelper` as opposed to `app/helpers/application.rb` being named `Helper::Application` 

Existing naming wasn't bad but I wanted to bring the view models and presenters up to the same level as `models` in the dir structure rather than hiding them in there.